docs/source/contributing.md

```diff
@@ -1,14 +1,30 @@
-Contributing to the Leapp project
-=================================
+# Contributing to the Leapp project
 
 First, thank you for taking your time to contribute to the project.
 
 The following is a set of guidelines for contributing effectively to the Leapp-related repositories
 hosted under the `OS and Application Modernization Group organization <https://github.com/oamg/>`_
 on GitHub.
 
-Submitting a Pull Request
-^^^^^^^^^^^^^^^^^^^^^^^^^
+## Code style guidelines
+
+Your code should follow our
+[Python Coding Guidelines](python-coding-guidelines.html)
```

Comment:
```suggestion
[Python Coding Guidelines](python-coding-guidelines)
```
Do not link to `.html`, there was a problem with that. See https://github.com/oamg/leapp/pull/602. Update the other references as well.